src/css/blog-post.module.scss

```diff
@@ -310,9 +318,17 @@
 // Responsive Design
 
 @media screen and (max-width: 768px) {
+  .blogMainImage {
+    width: 93%;
+    height: 12rem;
```

Comment:
Setting both dimensions may distort the image in some cases. if my eye doesn't trick me, I believe the mobile version distorts the blog image, can you check?

Reply:
Yes, the same is with the home page is in mobile view, and the solution in that case was that the image has a fit-object: cover and nothing else. Also, it is important to add that the images will change to something simpler. Thus, it will look better.

Heavy picture:
![Captura desde 2023-04-05 14-58-18](https://user-images.githubusercontent.com/107500715/230164351-7ac00910-958a-4d07-8963-6a36f243042c.png)

Simple picture:
![Captura desde 2023-04-05 14-56-55](https://user-images.githubusercontent.com/107500715/230164239-7ab754b6-cf7c-4784-b330-fbca4c33ce21.png)

Does it answer your question?

Reply:
Good! it does